src/FormGroup.js

```diff
@@ -26,6 +28,7 @@ FormGroup.defaultProps = {
 FormGroup.propTypes = {
   standalone: React.PropTypes.bool,
   hasFeedback: React.PropTypes.bool,
+  bsSize: React.PropTypes.oneOf(['small', 'medium', 'large']),
```

Comment:
To avoid confusion when those classes don't show up because the standalone prop is used. Can you do a custom prop type validation here to check if the standalone prop is true when using this and warn if so?